great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -80,7 +79,6 @@ class _PandasDatasource(Datasource):
     asset_types: ClassVar[List[Type[DataAsset]]] = list(_ASSET_MODELS.values())
 
     # instance attributes
-    type: str = pydantic.Field("_pandas")
```

Comment:
This field serves no purpose because this class is not meant to be instantiated. Was only defined to prevent a crash during `Metadatasource.__new__` registration. No that `Metadatasource` skips registration for `_Datasources` we can remove it.